fairlearn/experimental/enable_metric_frame_plotting.py

```diff
@@ -0,0 +1,17 @@
+"""Enables Metric Frame Plotting
+
+The API and results of these estimators might change without any deprecation
+cycle.
+
+Importing this file dynamically sets the
+:class:`~metrics._error_plotter.plot_metric_frame` as an attribute of the
+`MetricFrame` module::
```

Comment:
`MetricFrame` is a class, not a module, `metrics` is a module (or I'm still misunderstanding Python terminology which is possible!).

Reply:
I'm pretty sure that's right - at the very least `MetricFrame` is a class.

Reply:
Sorry this is irrelevant now that we moved `plot_metric_frame` to a separate file instead of being a function under MetricFrame